src/bp/lang-server/service/index.ts

```diff
@@ -41,6 +55,20 @@ export default class LanguageService {
     this._models = this._getModels()
     const languages = Object.keys(this._models)
 
+    if (languages.length > 0) {
+      const ramInfos = checkEnoughRam(languages.length, this.dim)
+
+      if (ramInfos.prediction < ramInfos.total - ramInfos.used) {
+        this.logger.info(`Lang server will take ~${ramInfos.prediction}Gb or more`)
```

Comment:
There's no need to tell the user he has enough memory.

Reply:
Hmmm right, if the user has enough memory, it'll run, if not it'll crash. What we can do though is to log something if the user doesn't have enough mem. I suggest that required mem is computed with language model size + a buffer of let's say 20% of the model size because of all the caching. something like that :

```
requiredMem = modelSize + .2 * modelSize
if availableMem < requiredMem
  log "Not enough memory to start lang server, free mem and try again"
  process.exit(1)
```